feature/policy_forwarding/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,174 @@
+# PF-1.6: Policy forwarding based GUE tunnel decap to default and non-default network-instance
+
+## Summary
+This test ensures NOS is able to host multiple VRFs, perform GUE tunnel DECAP in the default VRF and also allows for gradual traffic migration from Default to Non-Default VRF using VRF selection policy.
+
+
+## Test environment setup
+
+### Topology
+Create the following connections:
```

Comment:
ATE arent' modeled as a single device with multiple ports.  Each ate port is independent.  So you can just call the ATE ports,


ATE-Port1
ATE-Port2

(no need to refer to ATE1-Port1, ATE2-Port1)

Reply:
_done_